Remotion/Web/Development.Analyzers/Web.Development.Analyzers.csproj

```diff
@@ -0,0 +1,27 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <LangVersion>9</LangVersion>
+    <Nullable>enable</Nullable>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <DevelopmentDependency>true</DevelopmentDependency>
+    <IncludeBuildOutput>false</IncludeBuildOutput>
+    <NoPackageAnalysis>true</NoPackageAnalysis>
+    <AssemblyName>Remotion.Web.Development.Analyzers</AssemblyName>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.CodeAnalysis.CSharp.Workspaces" Version="3.8.0" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <None Include="$(OutputPath)\$(AssemblyName).dll" Pack="true" PackagePath="analyzers/dotnet/cs" Visible="false" />
```

Comment:
This type if include means we have the assembly show up in the visual studio tree, which isn't best practice. There is the option of adding the assembly directly to the package outout, see https://github.com/NuGet/Home/issues/4704#issuecomment-315894224 or https://github.com/NuGet/Home/issues/4704#issuecomment-315915898. I haven't done a deep dive into the pro/cons of the two versions, but the first one that works should be fine.

I would recommend we consider setting up a dedicated Analyzers.targets proejct type, though. We could do this in a separate task and refactor here when we do the task. depends on how much time is sensible to invest now vs later.

Reply:
> This type if include means we have the assembly show up in the visual studio tree,

That's what `Visible="false"` is for.

Reply:
ah, I.C. Didn't find notice that one. Verify interesting. It may offend my puristic views but I can certainly see why this makes things easier to handle so it's probably best that I start liking :D Where did you learn about `Visible`?

Reply:
Can't remember but it's handy